raisinbread/README.md

```diff
@@ -49,13 +61,7 @@ cat SQL/0000-00-00-schema.sql \
     raisinbread/RB_files/*.sql | mysql
 ```
 
-Note: to empty and delete all RaisinBread tables, use the following command in the 
-main LORIS root directory
-```bash
-cat raisinbread/instruments/instrument_sql/9999-99-99-drop_instrument_tables.sql \
-    SQL/9999-99-99-drop_tables.sql | mysql
-```
-
+If the tables were not deleted or created properly, the above schemas can be sourced directly on the mysql command line.
```

Comment:
I don't understand what this sentence is trying to explain or instruct the user to do.

Reply:
I think it's in contrast to the `cat` command. 

the suggested command will stop running after the first failure while sourcing the script directly in the database will be more explicit and continues running the remainder of the commands after 1 has failed

Reply:
I think that might be clearer if it was in the paragraph above the code which currently says:

"The commands below assume that the current working directory is the main LORIS root
directory."

Something like:

"The commands below assume that the current working directory is the main LORIS root
directory. If the tables are not deleted or created properly, the schemas files can be sourced directly on the mysql command line."

(I can't suggest changes because it's not a line changed by the commit..)

Reply:
@AlexandraLivadas that makes sens to me, you can make the changes

Reply:
Sounds good!